.github/ISSUE_TEMPLATE/Third_party_addon.md

```diff
@@ -1,6 +1,7 @@
 ---
 name: '3rd party add-on '
 about: Tell us about a 3rd party add-on that we should add or update in the ZAP Marketplace
+labels: Type-Task

```

Comment:
Add-on?
Third party? (Though it wasn't how I originally envisioned it being attributed it'd still fit I guess)

Reply:
Done.